source/manual/howto-switch-default-branch.html.md

```diff
@@ -0,0 +1,26 @@
+---
+owner_slack: "#govuk-developers"
+title: How to switch a git default branch from master to main
+section: GitHub
+layout: manual_layout
+parent: "/manual.html"
+---
+
+These steps are for switching the default branch of a repo to `main`.
+
+1. Rename the `master` branch in GitHub to `main` for that repo. See [instructions from Github for renaming an existing default branch](https://github.com/github/renaming#renaming-existing-branches).
+1. Update the [Release app](https://release.publishing.service.gov.uk/applications/):
+  - Go to the app entry from the [listing](https://release.publishing.service.gov.uk/applications/).
+  - Hit the `Edit` tab.
+  - Switch under the dropdown for `GitHub repository default branch`.
+1. [Reset the Jenkins build number](https://docs.publishing.service.gov.uk/manual/test-and-build-a-project-on-jenkins-ci.html#fixing-the-build-number).
+1. Delete the master branch locally, otherwise you might rebase the wrong branch by mistake!
```

Comment:
GitHub includes some instructions on how to do this that it might be useful to link to: https://docs.github.com/en/github/administering-a-repository/renaming-a-branch#updating-a-local-clone-after-a-branch-name-changes

Rather than deleting the `master` branch locally, it renames it to `main` and keeps it up to date with origin.

Reply:
Ooooh! 😍